ops/storage-ddls/cassandra-setup.cql

```diff
@@ -9,24 +9,39 @@ USE fortis;
 
 DROP TYPE IF EXISTS computedgender;
 CREATE TYPE computedgender (
-    male_mentions int,
-    female_mentions int
+    male_mentions bigint,
+    female_mentions bigint
+);
+
+DROP TYPE IF EXISTS sentiment;
+CREATE TYPE sentiment (
```

Comment:
Can this just be a single number for sentiment as per the advantages described in #90?